cockatrice/src/dlg_settings.cpp

```diff
@@ -438,6 +442,7 @@ void UserInterfaceSettingsPage::retranslateUi()
     annotateTokensCheckBox.setText(tr("Annotate card text on tokens"));
     animationGroupBox->setTitle(tr("Animation settings"));
     tapAnimationCheckBox.setText(tr("&Tap/untap animation"));
+    idleClientTimeOutCheckBox.setText(tr("Disconnect from server if idle for 1 hour"));
```

Comment:
Hard-coded numbers should not be part of the translation, especially if they *could* change.
Is it possible to extract the value from settings as a variable?

(In case this is really just a temporary change and will be addressed/solved very soon - I don't care too much...)